### PR TITLE
Updating to new contracts

### PR DESCRIPTION
**Pull request by josojo**
Step one1: just use the new dex-contracts code and the new rinkeby deployment

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ matrix:
         directories:
           - dex-contracts/node_modules/
           - $HOME/.npm
-      script: cd dex-contracts && npm ci
+      script: cd dex-contracts && yarn install --frozen-lockfile
```
Comment (fleupold) [inline .travis.yml]: Let's do that switch in a separate PR can we?

File: README.md [hunk 2]
```diff
@@ -10,32 +10,33 @@ This repository contains the backend logic for the dfusion exchange based on the
   <img src="documentation/architecture.png" alt="dex-services architecture" width="500">
 </p>
 
-The *Event Listener* registers for certain EVM events via the [Gnosis Trading DB](https://github.com/gnosis/pm-trading-db).
+The _Event Listener_ registers for certain EVM events via the [Gnosis Trading DB](https://github.com/gnosis/pm-trading-db).
```
Comment (fleupold) [inline README.md]: Do we really want these changes? Would you mind turning off autoformat on save for the README file?

File: .travis.yml (hunk 1 above)
Comment (josojo) [reply .travis.yml]: No, otherwise the build is failing!

Comment (josojo) [reply .travis.yml]: because the package-lock is no longer available and then npm ci fails

Comment (nlordell) [reply .travis.yml]: When we update to a new `dex-contracts` we also have to update to use `yarn` here.

File: README.md [hunk 3]
```diff
@@ -51,12 +52,13 @@ The project may work with other versions of these tools but they are not tested.
...
-Clone the repository, its submodule, and run the container
+Clone the repository, its submodule, and run the container:
+
 ```bash
 git clone git@github.com:gnosis/dex-services.git
 cd dex-services
 git submodule update --init
-cd dex-contracts 
+cd dex-contracts
 npm ci && npx truffle compile
```
Comment (nlordell) [inline README.md]: This should probably be updated to use `yarn` or `yarn install` (they should be equivalent) instead of `npm ci`.

Comment (bh2smith) [reply README.md]:
```suggestion
yarn install && npx truffle compile
```
